rs/ic_os/manifest_tool/BUILD.bazel

```diff
@@ -0,0 +1,38 @@
+load("@rules_rust//rust:defs.bzl", "rust_binary", "rust_test")
+
+package(default_visibility = ["//rs:ic-os-pkg"])
+
+DEPENDENCIES = [
+    # Keep sorted.
+    "@crate_index//:anyhow",
+    "@crate_index//:clap",
+    "@crate_index//:regex",
+    "@crate_index//:tempfile",
+    "@crate_index//:walkdir",
+]
+
+MACRO_DEPENDENCIES = [
+    # Keep sorted.
+]
+
+ALIASES = {}
+
+rust_binary(
+    name = "manifest_tool",
+    srcs = glob(["src/**/*.rs"]),
+    aliases = ALIASES,
+    crate_name = "manifest_tool",
+    edition = "2021",
+    proc_macro_deps = MACRO_DEPENDENCIES,
+    target_compatible_with = [
```

Comment:
@andrewbattat this needs to be removed also for bazel to build it on other platforms

Reply:
Removed and successfully tested on my local Mac. Thank you!